api/backend_test.go

```diff
@@ -870,6 +870,7 @@ func TestLoginAccount(t *testing.T) {
 
 	require.NotEmpty(t, accounts[0].KeyUID)
 	require.Equal(t, acc.KeyUID, accounts[0].KeyUID)
+	require.True(t, acc.HasAcceptedTerms)
```

Comment:
I'm a bit cofused, why would `HasAcceptedTerms` be `true` just right after `CreateAccountAndLogin`/`Logout`? 🤔

Reply:
Good catch, this assertion should be a few lines above, right after the account is created. For the reason you identified in another comment, we want to check the flag is set to true when the first account is created, meaning if the user could create an account it implies the client requested the user accept the terms.

Reply:
Already pushed the fix.